lib/Sema/TypeCheckAttr.cpp

```diff
@@ -3182,20 +3182,15 @@ static AbstractFunctionDecl *findAbstractFunctionDecl(
 
   // Perform lookup.
   LookupResult results;
+  // If `baseType` is not null but `lookupContext` is a type context, set
```

Comment:
Note: this change is mostly NFC. I failed to find a minimal reproducer for the fixed issue TF-1090 in https://github.com/apple/swift/pull/28935, though I didn't try very hard. I think no test on `master` is fine.